articles/connections/database/custom-db/templates/create.md

```diff
@@ -439,83 +414,66 @@ function create(user, callback) {
 
   connection.connect();
 
-  var query = "INSERT INTO users SET ?";
+  const query = 'INSERT INTO users SET ?';
```

Comment:
In Heroku preview, code in code block is rendered strangely from here on (green, when it should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/create#mysql